test_agasc1p7.py

```diff
@@ -0,0 +1,47 @@
+
+import numpy as np
+import tables
+
+h5 = tables.openFile('agasc1p7.h5', mode='r')
```

Comment:
This file should be called (here) something like `agasc1p7_from_fits.h5`, and the other one should be `agasc1p7_from_jupyter.h5`.  Otherwise the heritage is very confusing.  You can do this with soft links and add a code comment describing where each file came from (first from the `create_agasc_h5.py` and second from the Jupyter notebook).

These docs won't be wasted effort since you will need to describe the file creation process in detail for the final AGASC 1.7 README (or wherever we decide that needs to go).